### PR TITLE
Add support second-level granularity to injectTimes 

### PR DESCRIPTION
**Pull request by yykcool**
## Description
**Linked issue**: None 

**Problem**
seconds from Datetime objects passed into injectTime setting is currently ignored by the date picker

**Changes**
added support for seconds, changed doc-site example to reflect it, and added 1 test as well.

also corrected `timeIntervals` to `intervals` on existing tests found in `inject_times_test.test.js` to account for the fact that `timeIntervals` does not exist as a parameter, and the correct one  should be `intervals` instead

## Contribution checklist
- [x] I have followed the [contributing guidelines](https://github.com/Hacker0x01/react-datepicker/blob/main/CONTRIBUTING.md).
- [x] I have added sufficient test coverage for my changes.
- [x] I have formatted my code with Prettier and checked for linting issues with ESLint for code readability.

### CODE REVIEW

**Review by martijnrusschen — CHANGES_REQUESTED** [review]
Please fix the test case comments

**Comment by yykcool** [comment]
comments are addressed

File: test/inject_times_test.test.js [hunk 1]
```diff
@@ -82,4 +83,31 @@ describe("TimeComponent", () => {
       "12:03 AM",
     ]);
   });
+
+  it.only("should support hours, minutes, and seconds", () => {
```
[the remark this answers is not in the document]
Comment (yykcool) [reply test/inject_times_test.test.js]: resolved

File: test/inject_times_test.test.js [hunk 2]
```diff
@@ -82,4 +83,31 @@ describe("TimeComponent", () => {
...
+          utils.addHours(utils.addMinutes(utils.addSeconds(today, 1), 30), 1),
+        ]}
+      />,
+    );
+
+    const injectedItems = container.querySelectorAll(
+      ".react-datepicker__time-list-item--injected",
+    );
+
+    console.log(Array.from(injectedItems).map((node) => node.textContent));
```
[the remark this answers is not in the document]
Comment (yykcool) [reply test/inject_times_test.test.js]: resolved

**Comment by martijnrusschen** [comment]
Can you make sure to merge with the latest main branch to upload the code coverage file?

**Comment by yykcool** [comment]
it seems like things are already up to date (tried to do a merge from the main branch), is there anything i might have missed?

**Comment by martijnrusschen** [comment]
can you try again?

**Comment by yykcool** [comment]
done

**Pull request merged**